appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -58,13 +59,20 @@ internal fun Modifier.outerPrimitiveStyle(
             .margin(style.getMargins())
             .styleShadow(style, isDark)
             .styleSize(style, matchParentBox)
-            .actions(id, gestureProperties, component.textDescription)
+            // Exclude actions for CustomComponent since those if needed should be triggered manually
+            .conditional(component !is CustomComponentPrimitive) { actions(id, gestureProperties, component.textDescription) }
```

Comment:
hmmn, want to check with Matt on this part. I thought maybe there was an idea of allowing actions to be set up on the component like normal, on tap, handled here. Then also having the component be able to trigger some things internally.

Reply:
sure we can talk about this, he's done similar behavior on iOS (I confirmed before doing it myself), so they currently can define tap actions from the builder, but they decide where that is triggered by calling actionsController.triggerActions()

Reply:
got it, ok, thanks for clarifying - i had not kept up on that aspect.